tests/template_test.py

```diff
@@ -0,0 +1,39 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import BaseTestCase, interfaces, context, irrelevant
+from utils.warmups import default_warmup
+
+# This warmup task runs to make sure containers are ready to receive requests
+context.add_warmup(default_warmup)
```

Comment:
You can remove this line, it's only for dedicated scenarios

Reply:
Maybe it's good to leave it in but comment it out? and explain what it is for?